integreat_cms/static/src/css/style.scss

```
@@ -561,7 +561,61 @@ datalist option {
 }
 
 /* Special word break for feedback list */
-.feedback-comment-full {
-    @apply break-words whitespace-pre-line;
-    word-break: break-word;
+.feedback-entry {
+    display: fleX;
+    flex-wrap: nowrap;
+    align-items: center;
+
+    span {
+        overflow: hidden;
+        text-overflow: ellipsis;
+        white-space: no-wrap;
+    }
+
+    .clickable-icon {
+        display: inline-flex;
+        cursor: pointer;
+        :hover {
+            transform: scale(1.25);
+        }
+    }
+
+    .copy {
+        @apply text-green-500;
+        animation: copy-feedback 0s 3s forwards;
+        visibility: hidden;
+    }
+
+    @keyframes copy-feedback {
+        to {
+            visibility: hidden;
+        }
+    }
+
+    a {
+        padding: 0 0.1rem;
+    }
+
+    &.active {
+        flex-wrap: wrap;
+
+        span {
+            @apply break-words whitespace-pre-line;
+            word-break: break-word;
+            overflow: visible;
+            white-space: initial;
```

```suggestion
            @apply break-all whitespace-normal overflow-visible;
```
I think the `@apply` notation is a bit more concise